roles/ironic-common/tasks/main.yml

```diff
@@ -36,16 +28,16 @@
   file: dest=/var/log/ironic state=directory owner=root group=ironic mode=0770
 
 - name: ironic config
-  template: src=etc/ironic/ironic.conf dest=/etc/ironic/ironic.conf mode=0640 owner=root group=ironic
+  template: src=etc/ironic/ironic.conf dest=/etc/ironic/ironic.conf mode=0640 
+            owner=root group=ironic
 
 - name: ironic policy config
-  template: src=etc/ironic/policy.json dest=/etc/ironic/policy.json mode=0640 owner=root group=ironic
+  template: src=etc/ironic/policy.json dest=/etc/ironic/policy.json mode=0640 
+            owner=root group=ironic
 
 - name: ironic paste config
-  template: src=etc/ironic/rootwrap.conf dest=/etc/ironic/rootwrap.conf mode=0640 owner=root group=ironic
-
-- name: ironic rootwrap
-  command: rsync -avh {{ ironic.path }}/etc/ironic/rootwrap.d /etc/ironic
+  template: src=etc/ironic/rootwrap.conf dest=/etc/ironic/rootwrap.conf 
+            mode=0640 owner=root group=ironic
```

Comment:
Looks like this was probably a think-o on the first submission, but the rootwrap.conf is not paste config. The task is named wrong, might as well fix it while you're here.